src/game/Laptop/BobbyR.cc

```diff
@@ -444,7 +446,7 @@ static void InitBobbyRayNewInventory(void)
 	UINT16	usBobbyrIndex = 0;
 
 
-	memset( LaptopSaveInfo.BobbyRayInventory, 0, sizeof(STORE_INVENTORY) * MAXITEMS);
+	std::fill_n(LaptopSaveInfo.BobbyRayInventory, static_cast<size_t>(MAXITEMS), STORE_INVENTORY{});
```

Comment:
Wont that result in the array filled with references to the same `STORE_INVENTORY{}` struct?

Reply:
Nope, `LaptopSaveInfo.BobbyRayInventory` is an array of concrete structures, not references.
`STORE_INVENTORY{}` is a temporary default/zero-filled structure that is assigned (copied) to each element of the array.

Reply:
Thanks for clarification, I just wanted to make sure...